property/index.js

```diff
@@ -141,22 +142,11 @@ module.exports = yeoman.generators.Base.extend({
     if (this.required) {
       def.required = true;
     }
+
     if (this.defaultValue) {
-      if (this.type === 'boolean'){
-        if (['true', '1', 't'].indexOf(this.defaultValue) !== -1 ){
-          def.default = true;
-        } else {
-          def.default = false;
-        }
-      } else if (this.defaultValue === 'uuid' || this.defaultValue === 'guid'){
-         def.defaultFn = this.defaultValue;
-      } else if ((this.type === 'date' || this.type === 'datetime') &&
-                  this.defaultValue.toLowerCase() === 'now'){
-         def.defaultFn = 'now';
-      } else {
-         def.default = this.defaultValue;
-      }
+      def = coerceDefault(def, this.defaultValue);
```

Comment:
IIUC, `coerceDefault` takes `def` object as an input and returns a new/modified `def` object as the return value. I find it unnecessary complex, can we please simplify to:

```js
def.default = coerceDefaultValue(this.defaultValue, this.type);
```